PowerEditor/src/WinControls/OpenSaveFileDialog/CustomFileDialog.cpp

```diff
@@ -51,6 +50,7 @@ namespace // anonymous
 
 	static const int IDC_FILE_CUSTOM_CHECKBOX = 4;
 	static const int IDC_FILE_TYPE_CHECKBOX = IDC_FILE_CUSTOM_CHECKBOX + 1;
+	static const TCHAR TEMP_OK_BUTTON_LABEL[] = _T("Save");
```

Comment:
Following C++14+, this should probably be `static constexpr const...`.

Reply:
In this context, I think it doesn't matter.
The compiler generates the same assembly.

If I were to declare this as a static member of a class, then it makes more sense. 
Since I could initialize `constexpr` in place while for a `static const` I need an initializer in the `.cpp`.

Reply:
Sure, I just make it a habit nowadays to use `constexpr` wherever it is relevant instead of just `const` alone.

Many things will generate the same machine code output, but will have slightly different semantics, but in general I keep it `constexpr` because it encourages me to use it more often.